### PR TITLE
Site: Add Tencent and TBDS logo in powered-by image

### PR DESCRIPTION
**Pull request by XuQianJin-Stars**
Update the powered-by image, add Tencent and TBDS logo on it.

### CODE REVIEW

**Comment by yanlin-Lynn** [comment]
The size of the powered-by.png increase a lot, does it matter?

**Comment by chunweilei** [comment]
> The size of the powered-by.png increase a lot, does it matter?

I think it's ok.

**Comment by XuQianJin-Stars** [comment]
hi, @yanlin-Lynn @chunweilei  Any other questions?

File: site/_docs/powered_by.md [hunk 1]
```diff
@@ -137,3 +137,7 @@ It uses Calcite to transparently rewrite queries to use materialized views.
 is a SQL interface to a
 <a href="https://tinkerpop.apache.org/">Apache TinkerPop</a>-enabled
 graph database.
+
+### TBDS
+
+[TBDS](https://cloud.tencent.com/product/tbds) (Tencent Big Data Suite) is a reliable, safe, and easy-to-use big data processing platform based on Tencent's years of massive data processing experience. With the help of TBDS, you can choose a suitable big data analysis engine and corresponding real-time data development, offline data development, and algorithm development services to build your data warehouse in public, private, and non-cloud environments. Big data application services such as user portraits, precise recommendations, and risk management.
```
Comment (DonnyZone) [inline site/_docs/powered_by.md:143]: The paragraph is too long. We'd better simplify it and focus on technique part.

Comment (hsyuan) [reply site/_docs/powered_by.md:143]: agree. see MaxCompute as example

Comment (XuQianJin-Stars) [reply site/_docs/powered_by.md:143]: well let me change it

File: site/_docs/powered_by.md [hunk 2]
```diff
@@ -137,3 +137,7 @@ It uses Calcite to transparently rewrite queries to use materialized views.
 is a SQL interface to a
 <a href="https://tinkerpop.apache.org/">Apache TinkerPop</a>-enabled
 graph database.
+
+### TBDS
+
+[TBDS](https://cloud.tencent.com/product/tbds) (Tencent Big Data Suite) is Big data computing and analysis platform, it uses calcite for query analysis.
```
Comment (DonnyZone) [inline site/_docs/powered_by.md:143]: How about ".... is a platform for big data storage and processing. It uses calcite for query analysis."?

Comment (XuQianJin-Stars) [reply site/_docs/powered_by.md:143]: > How about ".... is a platform for big data storage and processing. It uses calcite for query analysis."?

well ,Let me change it.

Comment (hsyuan) [reply site/_docs/powered_by.md]: What does query analysis mean? Parser? or optimizer?

Comment (XuQianJin-Stars) [reply site/_docs/powered_by.md]: > What does query analysis mean? Parser? or optimizer?

SQL parsing and query optimization

**Comment by chunweilei** [comment]
Could you squash commits, @XuQianJin-Stars ?

**Comment by XuQianJin-Stars** [comment]
> Could you squash commits, @XuQianJin-Stars ?

well ,Thks .

**Pull request merged**